src/ECModel.jl

```diff
@@ -852,6 +853,279 @@ function split_financial_terms(ECModel::AbstractEC, profit_distribution=nothing)
+        CAPEX=CAPEX,
+        OPEX=OPEX,
+        OEM = Ann_Maintenance,
+        REP = Ann_Replacement,
+        RV = Ann_Recovery,
+        REWARD = Ann_energy_reward,
+        PEAK = Ann_peak_charges,
+        EN_SELL = Ann_energy_revenues,
+        EN_CONS = Ann_energy_costs,
+        year_set = year_set
+    )
+end
+
+"""
+    business_plan(ECModel::AbstractEC, profit_distribution)
+
+Function to describe the cost term distributions by all users for all years.
+
+Parameters
+----------
+- ECModel : AbstractEC
+    EnergyCommunity model
+- profit_distribution
+    Final objective function
+- user_set_financial
+    User set to be considered for the financial analysis
+
+Returns
+-------
+    The output value is a NamedTuple with the following elements
+    - df_business
+        Dataframe with the business plan information
+"""
+
+function business_plan(ECModel::AbstractEC,profit_distribution=nothing, user_set_financial=nothing)
+    gen_data = ECModel.gen_data
+    
+    project_lifetime = field(gen_data, "project_lifetime")
+    
+    if isnothing(user_set_financial)
+        user_set_financial = get_user_set(ECModel)
+    end
+
+    # Create a vector of years from 2023 to (2023 + project_lifetime)
+    gen_data = ECModel.gen_data
+    project_lifetime = field(gen_data, "project_lifetime")
+
+    business_plan = split_yearly_financial_terms(ECModel)
+    year_set = business_plan.year_set
+
+    # Create an empty DataFrame
+    df_business = DataFrame(Year = Int[], CAPEX = Float64[], OEM = Float64[], EN_SELL = Float64[], EN_CONS = EN_SELL = Float64[], PEAK = Float64[], REP = Float64[], 
+    REWARD = Float64[], RV = Float64[])
+    for i in year_set
+        Year = 0 + year_set[i+1]
+        CAPEX = sum(business_plan.CAPEX[i, :])
+        OEM = sum(business_plan.OEM[i, :])
+        EN_SELL = sum(business_plan.EN_SELL[i, :])
+        EN_CONS = sum(business_plan.EN_CONS[i, :])
+        PEAK = sum(business_plan.PEAK[i, :])
+        REP = sum(business_plan.REP[i, :])
+        REWARD = sum(business_plan.REWARD[i, :])
+        RV = sum(business_plan.RV[i, :])
+        push!(df_business, (Year, CAPEX, OEM,  EN_SELL, EN_CONS, PEAK, REP, REWARD, RV))
+    end
+
+    return df_business
+end
+
+"""
+    business_plan_plot(ECModel::AbstractEC, profit_distribution)
+
+Function to describe the cost term distributions by all users for all years.
+
+Parameters
+----------
+- ECModel : AbstractEC
+    EnergyCommunity model
+- df_business
+    Dataframe with the business plan information
+
+Returns
+-------
+    The output value is a plot with the business plan information
+"""
+
+function business_plan_plot(ECModel::AbstractEC, df_business=nothing,
+    xlabel="Year",
+    ylabel="Amount [k€]",
+    title="Business Plan Over 20 Years",
+    legend=:bottomright,
+    color=:auto,
+    xrotation=45,
+    bar_width=0.6,
+    grid=false,
+    framestyle=:box,
+    barmode=:stack,
+    scaling_factor = 0.001,
+    kwargs...)
+
+    if df_business === nothing

```

Comment:
Personally, I think we can do without the df_business input value for now.
However, in general, it is better to use the function `isnothing(df_business)` to check if a value is nothing